src/litdata/processing/functions.py

```diff
@@ -340,6 +342,7 @@ def optimize(
             Defaults to None.
         use_checkpoint: Whether to create checkpoints while processing the data, which can be used to resume the
             processing from the last checkpoint if the process is interrupted. (`Default: False`)
+        item_loader: The item loader used by the StreamingDataset if any.
```

Comment:
```suggestion
        item_loader: The item loader that will be used during loading in StreamingDataset. Determines
        	the format in which the data is stored and optimized for loading.
```
My proposal to change the description based on my understanding. 
To make it clear this loader is not used for loading during any of the optimization, but determines how we expect the dataset to be consumed.